components/AppMenu.jsx

```diff
@@ -302,6 +305,12 @@ class AppMenu extends React.Component {
             mousetrap(el).bind(this.props.appMenuShortcut, this.toggleMenu);
         }
     };
+    storeRigthMargin = (el) => {
+        if (this.props.menuCompact && el?.clientWidth > 0) {
+            const rightmargin = el.clientWidth - MiscUtils.convertEmToPx(11.5);
```

Comment:
What is 11.5em?

Reply:
Hello, 
11.5em  is the right margin used in CSS to hide a part of menu : 
https://github.com/qgis/qwc2/pull/308/files#diff-9af8fe3b349e7d4c7e12d7406c0dc4898315de424adb8a5410a783c774ac807bR60